app/mailers/application_mailer.rb

```diff
@@ -1,4 +1,4 @@
 class ApplicationMailer < ActionMailer::Base
-  default from: "from@example.com"
+  default from: ENV["host_email"]
```

Comment:
- thêm tên biến vào file application.yml.example em nhé
- tên biến môi trường in hoa. vd MAIL_HOST, ...

Reply:
vâng ạ